cmd/minishift/cmd/start.go

```diff
@@ -222,17 +225,34 @@ func runStart(cmd *cobra.Command, args []string) {
 			KubeConfigPath:   constants.KubeConfigPath,
 			OcPath:           ocPath,
 			AddonEnv:         viper.GetStringSlice(cmdUtil.AddOnEnv),
-			PublicHostname:   viper.GetString(configCmd.PublicHostname.Name),
+			PublicHostname:   configCmd.GetDefaultPublicHostName(ip),
+			SSHCommander:     sshCommander,
+			OcBinaryInsideVM: fmt.Sprintf("%s/oc", minishiftConstants.OcPathInsideVM),
 		}
 
 		clusterUpParams := determineClusterUpParameters(clusterUpConfig)
 		fmt.Println("-- OpenShift cluster will be configured with ...")
 		fmt.Println("   Version:", requestedOpenShiftVersion)
-		err = clusterup.ClusterUp(clusterUpConfig, clusterUpParams, &util.RealRunner{})
 
+		fmt.Printf("-- Copying oc binary from the image to VM ")
+		progressDots := progressdots.New()
+		progressDots.Start()
+		err = clusterup.CopyOcBinaryFromImageToVM(dockerCommander, minishiftConstants.GetOpenshiftImageName(requestedOpenShiftVersion), minishiftConstants.OcPathInsideVM)
+		if err != nil {
+			atexit.ExitWithMessage(1, fmt.Sprintf("Error copying the oc binary to %s: %v", minishiftConstants.OcPathInsideVM, err))
+		}
```

Comment:
I think we should print the location of the oc binary i.e. `/var/lib/minishift/bin`  for verbose logs.